### PR TITLE
Add emailOTP template to email-admin-config file

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
### Proposed changes in this pull request

Added the EmailOTP template to the email-admin-config.xml file

### Follow-ups

Please merge this PR prior to https://github.com/wso2/product-is/pull/7560

### CODE REVIEW

**Comment by somindatommy** [comment]
@ShanChathusanda93 There is an integration test related to Email Templates (`testGetAllEmailTemplateTypes`). Fix that as well.

In my opinion, you might have to open a migration issue as well.

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 1]
```diff
@@ -561,6 +562,57 @@
...
+                        </h1>
+                    </td>
+                </tr>
+                <tr>
+                    <td style="text-align: left; padding: 0px 50px 20px 50px;" valign="top">
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Hi,
+                        </p>
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Please use this one time password <b>{{OTPCode}}</b> to sign-in to your application.
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we update this to `One-Time PAssword`? WDYT?

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 2]
```diff
@@ -561,6 +562,57 @@
...
+                        </p>
+                    </td>
+                </tr>
+                <tr>
+                    <td style="text-align: left; padding: 30px 50px 50px 50px" valign="top">
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #505050; text-align: left;">
+                            Thanks,<br/>WSO2 Identity Server Team
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we add a `.` at the end? WDYT?

Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we change this to `Thank You,`?

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 3]
```diff
@@ -561,6 +562,57 @@
         </table>]]></body>
         <footer>---</footer>
     </configuration>
+    <configuration type="EmailOTP" display="EmailOTP" locale="en_US" emailContentType="text/html">
+        <subject>WSO2 - Email OTP</subject>
```
Comment (somindatommy) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Shall we add some more context to the subject? WDYT?

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 1 above)
Comment (ShanChathusanda93) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Changed.

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 2 above)
Comment (ShanChathusanda93) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Added.

Comment (ShanChathusanda93) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Changed.

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 3 above)
Comment (ShanChathusanda93) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Added.

**Pull request merged**